packages/zosfiles/CHANGELOG.md

```diff
@@ -2,14 +2,18 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
-
 ## Recent Changes
+
 - Enhancement: The `Copy.dataset` function now creates a new data set if the entered target data set does not exist. [#2349](https://github.com/zowe/zowe-cli/issues/2349)
+- Enhancement: Added the `maxLength` option to List SDK functions (`allMembers, dataSetsMatchingPattern, membersMatchingPattern`) to specify the maximum number of items to return. [#2409](https://github.com/zowe/zowe-cli/pull/2409)
```

Comment:
question for both changelog entries: Is (`allMembers, dataSetsMatchingPattern, membersMatchingPattern`) just one command/whatever, or is each item in the list its own individual thing? If the latter, each item should be enclosed in backticks: (`allMembers`, `dataSetsMatchingPattern`, `membersMatchingPattern`)

Reply:
Each item in the list is its own individual function. I'll adjust it to match the requested format, thanks

Reply:
Resolved in 469bc53